util/src/main/java/tc/oc/pgm/util/bukkit/MiscUtils.java

```diff
@@ -58,5 +58,7 @@ EntityChangeBlockEvent createEntityChangeBlockEvent(
 
   Key getSound(Sound constant);
 
+  Sound getSoundValue(String name);
```

Comment:
This is not really a "sound value", this is a sound.
 Consider renaming the previous method such that it's:

`Key getSoundKey(Sound constant);` 

and then renaming this one to

`Sound getSound(String name);`